Sources/ComposableArchitecture/Reducer/Reducers/OnChange.swift

```diff
@@ -42,13 +96,13 @@ extension Reducer {
   ///   - reducer: A reducer builder closure to run when the value changes.
   ///   - oldValue: The old value that failed the comparison check.
   ///   - newValue: The new value that failed the comparison check.
-  /// - Returns: A reducer that performs the
```

Comment:
This looked like an unfinished sentence. What did you want to say here? 😁  @stephencelis

Reply:
Not sure what @stephencelis was thinking in the moment, but something like "A reducer that performs the logic when the state changes." might work.